pkg/network/tracer/tracer_usm_linux_test.go

```diff
@@ -1223,7 +1224,7 @@ func testPostgresProtocolClassificationWrapper(enableTLS bool) func(t *testing.T
 func testPostgresProtocolClassification(t *testing.T, tr *Tracer, clientHost, targetHost, serverHost string, enableTLS bool) {
 	skippers := []func(t *testing.T, ctx testContext){skipIfUsingNAT}
 	if enableTLS {
-		t.Skip("TLS+Postgres classification tests are flaky")
+		flake.Mark(t)
```

Comment:
what does it mean?
when will the test run?

Reply:
@pducolin I guess you'll know best
I do see that on main we will still run the tests (even though the tests are flaky)
does it mean those tests will be reported as flaky in the datadog CI tests metrics?

Reply:
The tests still run, but they will not cause the job to fail. The same data is still reported to CI Visibility, so you can see failures/passes and whether or not a test is flaky.

Reply:
Then please exclude usm from the change, I don't want those tests to be reported as flaky as they will cause noise for us